src/main/java/com/onarandombox/MultiverseCore/commandtools/flags/FlagGroup.java

```diff
@@ -0,0 +1,132 @@
+package com.onarandombox.MultiverseCore.commandtools.flags;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * A group of flags.
+ */
+public class FlagGroup {
+    /**
+     * A builder for a flag group.
+     *
+     * @param name The name of the group.
+     * @return The builder.
+     */
+    public static @NotNull Builder builder(@NotNull String name) {
```

Comment:
Is the name always a command name? Seems like that is the intent. Maybe make that more clear here?

Reply:
yea there is, the name is just a unique identifier for getting a specific flaggroup, see the "mvcreate"

https://github.com/Multiverse/Multiverse-Core/blob/8b28cebedf4e7a46eb999af84ba54f9a5fd38f3b/src/main/java/com/onarandombox/MultiverseCore/commands/CreateCommand.java#L101

Reply:
Right... But if this is always only ever used for commands could we make it not an arbitrary string?

Reply:
![image](https://user-images.githubusercontent.com/30431861/216750358-e9f2d85f-e7c7-4892-aad3-b79a2c11fdd4.png)

mainly bcu to target a specific flaggroup for autocomplete, it is parsed as a string